CHARTER.md

```diff
@@ -309,6 +310,56 @@ o. Any issues that cannot be resolved by the TOB shall be referred to The Linux
    Foundation Executive Director for mediation, and, if necessary, for
    resolution by The Linux Foundation Board of Directors.
 
+p. OCI Working Groups.
+
+   - i. The OCI may establish one or more working groups (each, an “OCI Working
+     Group”) to experiment, validate, and define new OCI Specifications or
+     major revisions to existing OCI Specifications. Each OCI Working Group
```

Comment:
Major is pretty important here IMO. Working groups aren't the only way to make progress, specs already have maintainers and a change process.

Reply:
If the goals of the working group are completely in scope for an existing major release version the spec (for example changes to current 1.0 specs), then the working group is not necessary and should work with existing maintainers. A major revision of a spec (say image-spec 2.0) or a new spec should be considered very similar from a process perspective, no compatibility requirements with previous spec versions and separate set of maintainers who focus on the released major spec version.

What could be missing here is features which span multiple existing specs. There could be a working group which expects smaller changes in existing specs but not necessarily a separate released version as a result. In those cases the working group is expected to have maintainers of those specs as sponsors and the working group is dependent on those minor revisions getting released. As asked below, a working group is not intended to work around maintainers of a released spec version to push through a feature. If that needs to be more clear, then we can make it clear. Removing "major" here would necessitate that in my opinion.

Reply:
The intent isn't to work around any process. Rather, there are questions that can only be answered through some iteration, incubation, and some sort of validation. The idea is to provide a means for innovation and let the result be reviewed for whether it should be incorporated into existing or new specs. At the beginning, the group may think it's simple to put in an existing spec. Or, it may be complex requiring a new spec. It's through the iteration, incubation and validations that a proposal is formalized and likely piviots in some direction. 
If the end result is known at the beginning, then we don't need a working group.